scripts/utils/utils.lua

```diff
@@ -817,6 +817,11 @@ end
 ---@param t table
 ---@return integer|string, any
 function utils.randomEntryIdx(t)
+    if IsTableIpairsCompatible(t) then
```

Comment:
New globals will need to be added to `.vscode/settings.json` and `lua.sh`, I think